### ISSUE TITLE
Line breaks in alt text are not rendered

### ISSUE THREAD

**Issue by codeman38** [nickdenardis/social-visual-alt-text#40]
If the alt attribute contains a line feed character, this should be converted to <br> when displaying the text. Otherwise, multi-line alt text will get squashed into a single line, making it more difficult to read.

**Comment by nickdenardis** [comment]
@codeman38 
Great insight! 

Do you have a link to a post I can use as a test to show the before and after?

**Comment by codeman38** [comment]
@nickdenardis Here's one of the posts where I first noticed this: https://bsky.app/profile/fendersonia.bsky.social/post/3lilgq3wxxk2m

**Comment by nickdenardis** [comment]
@codeman38 
The updated code has been released and uploaded to the Chrome and Firefox app stores for review.

The fix will roll out as soon as they approve the update.

Thanks again for reporting this 😊
